shanoir-ng-datasets/src/main/java/org/shanoir/ng/dicom/web/service/DICOMWebService.java

```diff
@@ -296,48 +296,72 @@ private void sendMultipartRequest(HttpEntity entity) throws ShanoirException {
 		}
 	}
 
+	public void rejectExaminationFromPacs(String studyUID) throws ShanoirException {
+		String rejectURL = this.serverURL + "/" + studyUID;
+		if (wadoURLHandler.isWADO_URI(rejectURL)) {
```

Comment:
this if does not make sense anymore, as your rejectURL is always WADO-RS, as you build it yourself, please remove, the StudyInstanceUIDHandler does manage this part now for you normally, he extracts the StudyInstanceUID either from WADO-URI (old) or from WADO-RS (new) paths